Corona-Warn-App/src/androidTest/java/testhelpers/Junit5FakeAnnotations.kt

```diff
@@ -0,0 +1,16 @@
+@file:Suppress("PackageDirectoryMismatch")
+
+package org.junit.jupiter.api
+
+/**
+ * Allows us to share common base test classes between unit and instrumentation tests.
+ * Otherwise the compiler would complain about an unknown annotation class.
+ */
+@Target(
+    AnnotationTarget.ANNOTATION_CLASS,
+    AnnotationTarget.FUNCTION,
+    AnnotationTarget.PROPERTY_GETTER,
+    AnnotationTarget.PROPERTY_SETTER
+)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class AfterAll
```

Comment:
The compiler should complain because it is not part of `androidTest` dependencies.
We can remove this annotation , as it is shown as not used anywhere.

Please add to Android test and it will work:
 `androidTestImplementation "org.junit.jupiter:junit-jupiter-api:$jupiter_version"`

you will have to execute some License
```
        exclude 'META-INF/LICENSE.md'
        exclude 'META-INF/LICENSE-notice.md'
```

Reply:
I'm aware, this was on purpose.

Currently it's only `@AfterAll` we need to "fake", instead of  adding the whole jUnit5 dependency to instrumentation tests.
InstrumentationTests don't support jUnit5, if we add the full dependency, it also includes the jUnit5 `@Test` , which then means if a developer accidentally imports the wrong `@Test`. Tests annotated the wrong `@Test` would just be ignored and not run, but there will be no error.

How can we prevent this source of accidents if we include the full jUnit5 dependency?

Reply:
Then we should create a different `BaseTest` in UI package which handles Junit4 annotations. I don't see why we should mix them that way if it is problematic

Reply:
I liked the idea of sharing a base class but don't feel strongly about it. I've split up the base class into one for unit and one for instrumentation tests.